packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -64,7 +64,6 @@ export class InsuredBridgeL1Client {
   private instantRelays: { [key: string]: { [key: string]: InstantRelay } } = {}; // L1TokenAddress=>{depositHash, realizedLpFeePct}=>InstantRelay.
 
   private firstBlockToSearch: number;
-  private web3: Web3;
```

Comment:
I made a small clean up here. For some reason we diverged on how we declared web3 instances. within the clients we should _not_ refer to this as `web3` but rather define the network (l1/l2) to make it very clear in the code what we are talking to/about.